MergerLogicUnitTests/DataTypes/FSTest.cs

```diff
@@ -118,7 +118,8 @@ public void TileExists(bool isBase, Coord cords, bool isOneXOne, GridOrigin orig
                     .Returns<int, int, int>((z, x, y) => z == 2);
             }
 
-            var fsSource = new FS(this._pathUtilsMock.Object, this._serviceProviderMock.Object, "test", 10, isOneXOne, isBase, origin);
+            Grid grid = isOneXOne ? Grid.OneXOne : Grid.TwoXOne;
```

Comment:
the isOneXOne parameter can be replace to Grid parameter instead if you think there might be additional grids/ it isn't used as Boolean

this comment apply for every test

Reply:
Currently there isn't a plan to support additional grids.